packages/vuetify/src/components/VOverlay/locationStrategies.ts

```diff
@@ -77,14 +77,13 @@ export function useLocationStrategies (
   const updateLocation = ref<(e: Event) => void>()
 
   let scope: EffectScope | undefined
-  watchEffect(async () => {
+  watchEffect(() => {
     scope?.stop()
     updateLocation.value = undefined
 
     if (!(IN_BROWSER && data.isActive.value && props.locationStrategy)) return
 
     scope = effectScope()
-    if (!(props.locationStrategy === 'connected')) { await nextTick() }
```

Comment:
I suppose it could technically break any custom locationStrategy, that's probably very uncommon though and I think it should always be handled by the strategy itself instead.